tools/mo/openvino/tools/mo/main.py

```diff
@@ -92,6 +93,11 @@ def main(cli_parser: argparse.ArgumentParser, framework=None):
     print('[ SUCCESS ] Generated IR version {} model.'.format(get_ir_version(argv)))
     print('[ SUCCESS ] XML file: {}'.format(model_path))
     print('[ SUCCESS ] BIN file: {}'.format(model_path.replace('.xml', '.bin')))
+
+    # Unmap and delete temporary IR files
+    del ngraph_function
+    TempFilesCleaner().clean()
```

Comment:
What if we have an exception (failed conversion), these temporal files will not be deleted.

Reply:
It is deleted for now because we decided to not to use `mmap()` in MO at all